examples/activities/buildvrt.js

```diff
@@ -1,14 +1,15 @@
 "use strict";
 
 var fs = require('fs'),
-    path = require('path');
+    path = require('path'),
+    util = require("util");
 
 var tmp = require('tmp');
 
-var output = require("./output"),
-    shell = require("./shell");
+var output = require("../../lib/output"),
+    shell = require("../../lib/shell");
 
-module.exports = function buildVRT(files, outputUri, callback) {
+module.exports = function buildVRT(files, outputUri, options, callback) {
```

Comment:
@mojodna not sure if you want to keep this change around, this adds an `options` argument that will break previous versions of the buildVRT. I'm not sure if there's a good way to handle default args in activities so that this wouldn't be a required arg...but since the arguments passed into `this.activity` are turned into an array and passed around, and then eventually appended to with a callback, it's currently required that you at least give an empty object.